spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -533,55 +534,52 @@ public void testIsDeletedColumnWithoutDeleteFile() {
   public void testPosDeletesOnParquetFileWithMultipleRowGroups() throws IOException {
     Assume.assumeTrue(format.equals("parquet"));
 
-    String tblName = "test_multiple_row_groups";
+    String tblName = "test3";
```

Comment:
Kinda prefer meaningful name, like `table_with_multiple_row_groups`

Reply:
The reason I changed it to `test3` in the original (3.3) PR was that in the base class, the tables were `test` and `test2`, so I used `test3` for this new table to be consistent!